static/images/integrations/logos/trello.svg

```diff
@@ -1 +1 @@
-<svg width="200" height="200" viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg"><title>trello-mark-blue</title><defs><linearGradient x1="50%" y1="0%" x2="50%" y2="100%" id="a"><stop stop-color="#0091E6" offset="0%"/><stop stop-color="#0079BF" offset="100%"/></linearGradient></defs><g fill="none" fill-rule="evenodd"><rect fill="url(#a)" width="200" height="200" rx="25"/><rect fill="#FFF" x="113" y="26" width="61" height="87.5" rx="12"/><rect fill="#FFF" x="26" y="26" width="61" height="137.5" rx="12"/></g></svg>
\ No newline at end of file
+<svg width="200" height="200" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient x1="50%" y1="0%" x2="50%" y2="100%" id="a"><stop stop-color="#0091E6" offset="0%"/><stop stop-color="#0079BF" offset="100%"/></linearGradient></defs><g fill="none" fill-rule="evenodd"><rect fill="url(#a)" width="200" height="200" rx="25"/><rect fill="#FFF" x="113" y="26" width="61" height="87.5" rx="12"/><rect fill="#FFF" x="26" y="26" width="61" height="137.5" rx="12"/></g></svg>
```

Comment:
@fredfishgames: Why did you change all of these `.svg` files? I don't think you needed to. I would suggest getting rid of these changes unless you have a good reason to include them. If so, they should be in a separate commit! :)